src/wings_edge_cmd.erl

```diff
@@ -180,6 +189,12 @@ connect(Es0, #we{id=Id}=We0, Acc) ->
     We = wings_edge:dissolve_isolated_vs(Vs, We2),
     {We,[{Id,Sel}|Acc]}.
 
+connect_slide(St1) ->
+    {St0,Sel} = wings_sel:mapfold(fun connect/3, [], St1),
+    St=wings_sel:set(Sel, St0),
+    wings_wm:psend(geom, {save_state,St}),
```

Comment:
You should not use psend() here,  psend should only be used to send messages between processes.
Also that will cause the undo order to wrong. Maybe remove that line?